org.civicrm.bootstrapcivicrm/js/civihr-popup.js

```diff
@@ -0,0 +1,62 @@
+/*
+Prevent Popups to overflow tables.
+*/
+CRM.$(function($) {
+  'use strict';
+
+  (function($) {
```

Comment:
You are putting in the same file both the plugin definition (line 7-28) and its implementation (line 30-59). I think you should separate the two